src/Microsoft.AspNetCore.Authentication.OpenIdConnect/Events/AuthorizationCodeReceivedContext.cs

```diff
@@ -17,29 +18,72 @@ public class AuthorizationCodeReceivedContext : BaseOpenIdConnectContext
         /// <summary>
         /// Creates a <see cref="AuthorizationCodeReceivedContext"/>
         /// </summary>
-        public AuthorizationCodeReceivedContext(HttpContext context, OpenIdConnectOptions options, AuthenticationProperties properties)
+        public AuthorizationCodeReceivedContext(HttpContext context, OpenIdConnectOptions options)
             : base(context, options)
         {
-            Properties = properties;
         }
 
         public AuthenticationProperties Properties { get; set; }
 
         /// <summary>
-        /// Gets or sets the 'code'.
+        /// Gets or sets the <see cref="JwtSecurityToken"/> that was received in the authentication response, if any.
         /// </summary>
-        public string Code { get; set; }
+        public JwtSecurityToken JwtSecurityToken { get; set; }
```

Comment:
Does that correspond to the identity token or the access token, that can possibly be a JWT token?